.eslintrc.js

```diff
@@ -1,31 +1,363 @@
 module.exports = {
```

Comment:
can we also format this entire file ? Right now the tab is 2, the change here changes the tab to 4